docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst

```diff
@@ -0,0 +1,104 @@
+List of available Apache Kafka Connect connectors
+=================================================
+
+The following Apache Kafka Connect connectors are currently available, and can
```

Comment:
I'd remove "are currently available", since it must be a given (we don't list those that are unavailable). then the sentence become way easier to read - 
"The following connectors can be used in any Aiven for Kafka services with Kafka Connect enabled."

Reply:
fixed